tsconfig.json

```
@@ -19,7 +19,7 @@
       }
     ],
     "paths": {
-      "@/*": ["./src/*"]
+      "@/*": ["./*"]
```

NextJS는 `@/public` 이런 식으로 import 하는 방식으로 public 폴더에 접근할 수 없지 않나요?
절대 경로 URL 형태로 `/logo`이런 식으로 접근하지 않나요?

---

기본적으로 NEXT가 정적 파일 서빙할 때에는 모듈 형식으로 불러오지 않기 때문에, 리터럴에서는 말씀하신 것 처럼 활용되는 것이 맞습니다.

하지만, 저희 프로젝트에서는 svgr 라이브러리를 활용하여 svg 파일을 리액트 컴포넌트 형식으로 가져오는 방식을 선택하고 있습니다.
이 경우 svg가 일반적인 소스코드처럼 취급되기 때문에 public/assets에 있는 path alias를 활용하기 위해 수정하였습니다!

```suggestion
import LogoSVG from '@/public/assets/logo.svg';
```

만약 저렇게 활용하지 않을 경우 svg를 활용하기 위해 public에 접근할 때 파일마다 상대경로를 활용해야 하기 때문에 추후 오류가 발생할 확률이 높다고 생각했기 때문입니당

---

오호 그렇군요..! 답변 감사합니다 ㅎㅎㅎ